spec/models/institution_spec.rb

```diff
@@ -5,8 +5,35 @@
 
   # validations
   it { should validate_presence_of(:name)  }
+  it { should validate_numericality_of(:max_line_edits)  }
   it { should validate_uniqueness_of(:name)  }
   it { should allow_value("correct-value").for(:slug)  }
   it { should_not allow_value("value with space").for(:slug)  }
 
+  context "when an institution is first created" do
+    let(:institution) { FactoryBot.build(:institution) }
+
+    context ".create_default" do
+      it "creates a transcript convention" do
+        expect do
+          institution.save
+        end.to change { institution.transcription_conventions.count }.by(8)
+      end
+    end
+  end
+
+  context "when saving a record" do
+    let(:institution) { FactoryBot.build(:institution, max_line_edits: 4,
+                                         min_lines_for_consensus: 0, min_lines_for_consensus_no_edits: 0) }
+
+    before(:each) do
+      institution.save
```

Comment:
I think it'd be better to use `FactoryBot.create(:institution)` instead of `FactoryBot.build(:institution)`, so you can get rid of this `before` block ;)